docs/modules/ROOT/pages/subscriptions/getting-started.adoc

```diff
@@ -1,7 +1,17 @@
 [[getting-started]]
 = Getting Started with Subscriptions
 
-To get started with subscriptions you need a GraphQL server with subscription capabilities.
+To get started with subscriptions you need a GraphQL server with subscription capabilities. You can enable them by passing the `subscriptions` feature to `Neo4jGraphQL`:
```

Comment:
Usually in docs we put new sentences in a separate line, though they will appear all in a same paragraph on the website

```suggestion
To get started with subscriptions you need a GraphQL server with subscription capabilities. 
You can enable them by passing the `subscriptions` feature to `Neo4jGraphQL`:
```